.circleci/config.yml

```diff
@@ -62,7 +62,7 @@ jobs:
       - checkout
       - *attach_workspace
       - run:
-          command: npm run test:snapshots
+          command: npm run test:snapshots && npm run test:interaction
```

Comment:
Not a change request, just discussion. If we use yarn throughout the project should we use yarn for all of our CI commands as well? I don't think it should make too much of a difference as modules are installed with yarn but just a though.